src/test/java/com/jcabi/log/VerboseProcessTest.java

```diff
@@ -477,4 +557,42 @@ public int decide(final LoggingEvent event) {
             return decision;
         }
     }
+
+    /**
+     * Logger appender that compiles a list of all LoggingEvents.
+     */
+    private class TestAppender extends AppenderSkeleton {
+        /**
+         * List of logging events.
+         */
+        private final List<LoggingEvent> logs = new ArrayList<LoggingEvent>(10);
+
+        /**
+         * Provides all captured logging events.
+         * @return Copy of log list
+         */
+        public final List<LoggingEvent> getLogs() {
+            return new ArrayList<LoggingEvent>(this.logs);
+        }
+
+        /**
+         * I couldn't get PMB to stop triggering a `UncommentedEmptyMethodBody`
+         * error. I'm hoping a return statement makes it happy.
+         */
+        @Override
+        public final void close() {
+            return;
```

Comment:
@dean-e-clark oh, I see - so can you just suppress PMD with annotation and remove this `return;`?